gio/tests/dbus_peer.rs

```diff
@@ -0,0 +1,169 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+#[test]
```

Comment:
Wouldn't moving this to test::run be enough? so you don't need the duplicate function :)  otherwise lgtm

Reply:
Indeed, this can be untangled considerably. Thanks :)